docs/contributors/native-mobile.md

```diff
@@ -1,8 +1,31 @@
+
+Intertwined with the web codepaths, the Gutenberg repo also includes the [React Native](https://facebook.github.io/react-native/) based mobile tree.
+
+## Running Gutenberg Mobile on Android and iOS
+
+For instructions on how to run the **Gutenberg Mobile Demo App** on Android or iOS, see [Getting Started for the React Native based Mobile Gutenberg](/docs/contributors/getting-started-native-mobile.md)
+
+Also, the mobile client is packaged and released via the [official WordPress apps](https://wordpress.org/mobile/). Even though the build pipeline is slightly different then the mobile demo apps and lives in its own repo for now ([here's the native mobile repo](https://github.com/wordpress-mobile/gutenberg-mobile)), the source code itself is taken directly from this repo and the "web" side codepaths.
+
 ## Native mobile specific files
+
 The majority of the code shared with native mobile is in the very same JavaScript module and SASS style files. In the cases where the code paths need to diverge, a `.native.js` or `.native.scss` variant of the file is created. In some cases, platform specific files can be also found for Android (`.android.js`) or iOS (`.ios.js`).
+
 ## Mind the mobile
-Our tooling isn't as good yet as we'd like to and it's hard to have a good awareness of those native mobile files. That means that contributors need to manually pay attention to update the native mobile files during code refactorings. For example, renaming a function or a prop should also be performed in the native modules too, otherwise, the mobile client will break. We are in the process of putting some more safeguards in place in PRs, but we're still far from done. Please bear with us and thank you in advance. ❤️🙇‍
+
+Our tooling isn't as good yet as we'd like to and it's hard to have a good awareness of those native mobile files. That means that contributors need to manually pay attention to update the native mobile files during code refactorings. For example, renaming a function or a prop should also be performed in the native modules too, otherwise, the mobile client will break. We have added some mobile specific CI tests as safeguards in place in PRs, but we're still far from done. Please bear with us and thank you in advance. ❤️🙇‍
+
+## Native mobile E2E tests in Continuous Integration
+
+If you encounter a failed Android/iOS test on your pull request, we recommend the following steps:
+
+1. Re-running the failed Github Action job ([guide for how to re-run](https://docs.github.com/en/actions/configuring-and-managing-workflows/managing-a-workflow-run#viewing-your-workflow-history)) - This should fix failed test the majority of the time as there are still a few randomly occurring failures in tests that are being actively worked on. See the following github issue for updated info on known failures: https://github.com/WordPress/gutenberg/issues/23949
```

Comment:
> This should fix failed test the majority of the time as there are still a few randomly occurring failures in tests that are being actively worked on.

This sentence could probably need some refinement. WDYT about this:

```
This should fix the failed test for the majority of times but there might still be randomly occurring failures in tests that are being actively worked on.
```

Reply:
Thanks, I see how I wrote it was a bit unclear. I meant to give a reason for why re-running the tests will fix it in most cases (_because_ there are currently randomly occurring failures), and used "as" to mean "because" which is a bit ambiguous. 

I made this sentence more clear here: https://github.com/WordPress/gutenberg/pull/23951/commits/c79ea93c0a1725abb463152d037b7f66d66ef261.

Reply:
Ah, I see. https://github.com/WordPress/gutenberg/commit/c79ea93c0a1725abb463152d037b7f66d66ef261 works better so, 👍